systems/notification/api-gateway/pkg/rest/router.go

```diff
@@ -118,20 +118,26 @@ func formatDoc(summary string, description string) []fizz.OperationOption {
 
 func (r *Router) sendEmailHandler(c *gin.Context, req *SendEmailReq) (message emailPkg.SendEmailResponse, err error) {
 	payload := emailPkg.SendEmailRequest{
-		To:      req.To,
-		Subject: req.Subject,
-		Body:    req.Body,
-		Values:  req.Values,
+		To:           req.To,
+		TemplateName: req.TemplateName,
+		Values:       make(map[string]string),
 	}
+
+	// Convert map[string]interface{} to map[string]string
+	for key, value := range req.Values {
+		if strValue, ok := value.(string); ok {
+			payload.Values[key] = strValue
+		}
+	}
+
 	res, err := r.clients.m.SendEmail(&payload)
```

Comment:
Also, integration tests in `github.com/ukama/ukama/systems/notification/mailer/test/integration/mailer_test.go`  are failing. Please, make sure you fix those.